packages/calcite-components/src/components/swatch-group/swatch-group.scss

```diff
@@ -0,0 +1,26 @@
+:host {
+  display: flex;
+}
+
+.container {
+  display: flex;
+  inline-size: 100%;
+  flex-wrap: wrap;
+  align-items: center;
+  gap: theme("spacing.2");
```

Comment:
All these gaps should be using tokens, right?

Reply:
Yes, though this should be the same underlying value.